_content/articles/discourses-on-the-establishments-of-mindfulness-quoted-in-samathadeva's-abhidharmakosopayika-tika_dhammadinna.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Discourses on the establishments of mindfulness (*smṛtyupasthānas*) quoted in Śamathadeva’s *Abhidharmakośopāyikā-ṭīkā*"
```

Comment:
Oops! This file broke the site. 🙁 Please: no special characters in file names. 😁